core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -55,28 +55,18 @@ type dbReceipt struct {
 	CreatedAt        time.Time
 }
 
-func DbReceiptFromEvmReceipt(evmReceipt *EvmReceipt) dbReceipt {
+func DbReceiptFromEvmReceipt(evmReceipt *evmtypes.Receipt) dbReceipt {
 	return dbReceipt{
-		ID:               evmReceipt.ID,
 		TxHash:           evmReceipt.TxHash,
 		BlockHash:        evmReceipt.BlockHash,
-		BlockNumber:      evmReceipt.BlockNumber,
+		BlockNumber:      evmReceipt.BlockNumber.Int64(),
 		TransactionIndex: evmReceipt.TransactionIndex,
-		Receipt:          *evmReceipt.Receipt,
-		CreatedAt:        evmReceipt.CreatedAt,
+		Receipt:          *evmReceipt,
 	}
 }
 
-func DbReceiptToEvmReceipt(receipt *dbReceipt) EvmReceipt {
-	return EvmReceipt{
-		ID:               receipt.ID,
-		TxHash:           receipt.TxHash,
-		BlockHash:        receipt.BlockHash,
-		BlockNumber:      receipt.BlockNumber,
-		TransactionIndex: receipt.TransactionIndex,
-		Receipt:          &receipt.Receipt,
-		CreatedAt:        receipt.CreatedAt,
-	}
+func DbReceiptToEvmReceipt(receipt *dbReceipt) *evmtypes.Receipt {
```

Comment:
Caps?
```suggestion
func DBReceiptToEVMReceipt(receipt *dbReceipt) *evmtypes.Receipt {
```

Reply:
Er I guess `EvmReceipt` is the type name, so maybe out of scope.

Reply:
yeah i think i'll stick with `Evm` for now - it's used that way in quite a few places